Apps/Motioneye/docker-compose.yml

```diff
@@ -0,0 +1,70 @@
+name: motioneye
+services:
+  app:
+    cpu_shares: 10
+    command: []
```

Comment:
Please remove properties with empty list. There's a known issue that this will break the AppStore.